src/lib/mina_base/transaction_status.ml

```diff
@@ -206,6 +213,10 @@ module Failure = struct
         "Incorrect_nonce"
     | Invalid_fee_excess ->
         "Invalid_fee_excess"
+    | Incorrect_verification_key (`Expected_verification_key expected_key) ->
+        (* TODO: how large is the verification key? is it ok to base58? *)
```

Comment:
we store verification keys as Base64 in the archive db, so might be nice to follow that here